app/src/main/java/com/github/polypoly/app/base/menu/lobby/GameLobby.kt

```diff
@@ -18,7 +23,7 @@ data class GameLobby(
     val name: String = "defaultName",
     val code: String = "defaultCode",
     val private: Boolean = false,
```

Comment:
Can these default values go away now that this class isn't itself stored in the DB?